### PR TITLE
Remove variant files from regression tests

### PR DESCRIPTION
**Pull request by mtuncer**
Partial fix for #275. Tests are modified so that 2 variant files can be removed. However, 1 variant file (multi_upsert) still stays.

### CODE REVIEW

File: src/test/regress/sql/multi_large_table_join_planning.sql [hunk 1]
```diff
@@ -36,13 +36,12 @@ WHERE
 	l_partkey = p_partkey AND
 	c_custkey = o_custkey AND
         (l_quantity > 5.0 OR l_extendedprice > 1200.0) AND
-        p_size > 8 AND o_totalprice > 10 AND
-        c_acctbal < 5000.0
+        p_size > 8 AND o_totalprice > 10.0 AND
+        c_acctbal < 5000.0 AND l_partkey < 1000
```
Comment (metdos) [inline src/test/regress/sql/multi_large_table_join_planning.sql]: Do we need to add "AND l_partkey < 1000" and drop LIMIT?

Comment (mtuncer) [inline src/test/regress/sql/multi_large_table_join_planning.sql]: yes. limit 30 was displayed as
'30'::bigint in pg95. where pg94 displayed 30:bigint. I could not find a way to make them output the same text.

**Comment by metdos** [comment]
Could you also add a note the commit similar to this PR note to communicate we have still one variant regression file? Other than this, I think you can :ship:

**Comment by mtuncer** [comment]
updated commit text, rebased, squashed

**Pull request merged**